packages/forma-36-website/src/images/github.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think, we probably could optimize SVG a liitle bit, but it's just a minor thing, maybe it's not worth it...